### PR TITLE
fix(searching): removed quotes from search text

### LINKED ISSUE #5189: autocomplete - remove ""
Captura de ecrã 2022-10-07, às 16 29 48

do we need this "" before we start to search?

### PR DESCRIPTION
**Pull request by ThomBos**
### What this PR does 📖
- 

### Which issue(s) this PR fixes 🔨
- Resolve #5189 

### Special notes for reviewers 🗒️
- 

### Additional comments 🎤
-

### CODE REVIEW

**Review by stavares843 — COMMENTED** [review]
Captura de ecrã 2022-10-17, às 13 01 55

strange still appears here

**Comment by ThomBos** [comment]
> Captura de ecrã 2022-10-17, às 13 01 55
> 
> strange still appears here

that issue can't be caused by the change on this PR 😂

**Comment by ThomBos** [comment]
better opening a new one for this

**Comment by stavares843** [comment]
i meant the "" 😅

**Comment by stavares843** [comment]
Captura de ecrã 2022-10-17, às 14 21 59

**Comment by ThomBos** [comment]
Schermata 2022-10-17 alle 15 26 35

my bad, i removed these "" instead... 😓

**Comment by ThomBos** [comment]
@stavares843 should be good now 👍

**Review by stavares843 — APPROVED** [review]

File: locales/en-US.js [hunk 1]
```diff
@@ -640,6 +640,7 @@ export default {
         'Select one or more of your friends to chat in groups or one-on-one.',
       no_friends_yet: 'No Friends Yet',
       no_friends_yet_text: "The world doesn't have to be so lonely.",
+      search: 'Searching for',
```
Comment (stavares843) [inline locales/en-US.js:643]: thanks for adding 🔨

**Review by molimauro — APPROVED** [review]

**Pull request merged**